ecosystem/sep-0012.md

```diff
@@ -6,8 +6,8 @@ Title: KYC API
 Author: Interstellar
 Status: Active
 Created: 2018-09-11
-Updated: 2021-05-28
-Version 1.6.3
+Updated: 2021-06-24
+Version 1.7.0
```

Comment:
💭 I'm not convinced this should be a minor version, but instead a patch version. This is fixing a bug we introduced, albeit a big bug. It's a bug that doesn't introduce new functionality. I'm fine with minor if you prefer since this is definitely a gray area.